README.md

```diff
@@ -10,6 +10,19 @@ qtc
 go build -o bin/ ./cmd/...
 ```
 
+### Docker
+```shell script
+docker build -t matrix-static .
+docker run -v $(pwd)/config.json:/opt/matrix-static/config.json -p 8000:8000 -it matrix-static
```

Comment:
```suggestion
docker run -v $(pwd)/config.json:/opt/matrix-static/config.json -p 8000:8000 -it matrix-static
```

Does `matrix-static` here need to be an absolute path or will $PATH be set correctly so this is fine?

Reply:
![image](https://user-images.githubusercontent.com/2403652/73285354-ce27f080-41ed-11ea-8a2e-8ed4e9bd0661.png)

for bind mounts it seems like you must do absolute paths on both sides :((